README.md

```diff
@@ -80,12 +80,9 @@ To run different web applications for diffent virtual hosts:
        --httpsKeyManagerType    = the SSL KeyManagerFactory type (eg SunX509, IbmX509). Default is SunX509
        --httpsPrivateKey        = this switch with --httpsCertificate can be used to run HTTPS with OpenSSL secret key
          / --httpsCertificate     file and the corresponding certificate file
-       --http2Port               = set the http2 listening port. -1 to disable, Default is disabled
-       --http2ListenAddress      = set the http2 listening address. Default is all interfaces         
+       --http2Port              = set the http2 listening port. -1 to disable, Default is disabled
+       --http2ListenAddress     = set the http2 listening address. Default is all interfaces         
        --controlPort            = set the shutdown/control port. -1 to disable, Default disabled
-
-       --handlerCountMax        = set the max no of worker threads to allow. Default is 40
```

Comment:
What will happen if these parameters are defined in startup scripts? IIUC they will be just ignored, right? Even in such case, it makes sense to explicitly document it in the changelog

Reply:
will generate this:
```
webroot: EnvVars.masterEnvVars.get("JENKINS_HOME")
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at Main._main(Main.java:344)
	at Main.main(Main.java:160)
Caused by: java.lang.IllegalArgumentException: Unrecognized option: --handlerCountMax-=100
	at winstone.cmdline.CmdLineParser.parse(CmdLineParser.java:52)
	at winstone.Launcher.getArgsFromCommandLine(Launcher.java:356)
	at winstone.Launcher.main(Launcher.java:327)
```

Reply:
maybe restore them and log a WARNING and sure I will add that to the changelog